app/scripts/views/base.js

```diff
@@ -888,15 +888,13 @@ define(function (require, exports, module) {
       }
 
       if (_.isFunction(handler)) {
-        var args = [].slice.call(arguments, 1);
-
         // If an `arguments` type object was passed in as the first item,
         // then use that as the arguments list. Otherwise, use all arguments.
         if (_.isArguments(args[0])) {
           args = args[0];
         }
 
-        return handler.apply(this, args);
+        return handler.call(this, ...args);
```

Comment:
Although I support using the rest operator in the function signature and eliminating the `slice`, using the spread operator and `call` here seems less optimal than just sticking with `apply` to me.

Better to continue leaning on the JS interpreter to apply the arguments for us, than whatever Babel transpiles the spread operator to. Am I being unreasonable?